test/libsolidity/semanticTests/structs/struct_memory_to_storage_function_ptr.sol

```diff
@@ -32,6 +32,6 @@ contract C {
 // compileViaYul: also
 // ----
 // f() -> 42, 23, 34, 42, 42
-// gas irOptimized: 111210
```

Comment:
Haha nice, this reduction is because a `PUSH1 87` is replaced by a `PUSH1 1` :)

Reply:
Here's the diff: https://gist.github.com/hrkrshnn/03381b12959e8f5709a6aadff7cd69da

The difference is also from inlining.